pandas/tests/groupby/test_categorical.py

```diff
@@ -1165,3 +1165,13 @@ def test_seriesgroupby_observed_apply_dict(df_cat, observed, index, data):
         lambda x: OrderedDict([("min", x.min()), ("max", x.max())])
     )
     assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("code", [([1, 0, 0]), ([0, 0, 0])])
+def test_groupby_categorical_axis_1(code):
+    # GH 13420
+    df = DataFrame({"a": [1, 2, 3, 4], "b": [-1, -2, -3, -4], "c": [5, 6, 7, 8]})
+    grp = pd.Categorical.from_codes(code, categories=list("abc"))
```

Comment:
Can you call this `cat` instead of `grp`? The latter is typically the result of an actual groupby

Reply:
ok, changed! @WillAyd and thanks for taking a look!